dataprofiler/profilers/profile_builder.py

```diff
@@ -883,7 +908,7 @@ def save(self, filepath=None):
             pickle.dump(data, outfile)
 
         # Restore all data labelers
-        self._restore_data_labelers(data_labeler)
+        self._restore_data_labelers(data_labelers)
```

Comment:
could be `self._restore_data_labelers(unstruct_data_labeler, struct_data_labeler)`